### PR TITLE
Update styleguide with regards to import sorting

### PR DESCRIPTION
**Pull request by do3cc**
This is the end after a long discussion from here:
https://github.com/plone/plone.api/issues/187

### CODE REVIEW

**Comment by do3cc** [comment]
Oh, for the record, the discussion did not end in universal agreement and no formal voting.
It looks like the majority of people are in favor of the pull request in the discussion. But more importantly the arguments for case insensitive sorting weren't refuted while the arguments for case sensitive sorting were.

File: develop/styleguide/python.rst [hunk 1]
```diff
@@ -309,23 +309,21 @@ About imports
...
 
     # GOOD
     from __future__ import division
     from Acquisition import aq_inner
+    from Products.CMFCore.WorkflowCore import WorkflowException
+    from Products.CMFCore.interfaces import ISiteRoot
     from plone.api import portal
     from plone.api.exc import MissingParameterError
-    from Products.CMFCore.interfaces import ISiteRoot
-    from Products.CMFCore.WorkflowCore import WorkflowException
```
Comment (gforcada) [inline develop/styleguide/python.rst:328]: Aren't you sorting here as case sensitive while above saying it should  be case insensitive? :-) so the example was already ok.

Comment (do3cc) [inline develop/styleguide/python.rst:328]: Got me, I always mix it up. But if you followed the discussion, you know what I meant ;-)

**Comment by do3cc** [comment]
I'll drop this one

**Pull request closed without merging**